src/main/webapp/dev/js/main/instructorCourseEnrollPage.js

```diff
@@ -212,6 +219,40 @@ function addEnrollErrorMessages(enrollErrorLines) {
     });
 }
 
+/**
+ * Triggers an AJAX request to retrieve the enroll status.
+ * Does the necessary post processing after the state of the AJAX request is returned.
+ */
+function triggerAndProcessAjaxSaveAction() {
```

Comment:
Naming is not clear. Is it enroll or save. And save/enroll what? Also general words like trigger and process are not very useful.